app/routes/events/components/EventEditor/index.js

```diff
@@ -255,7 +257,26 @@ function EventEditor({
                 normalize={v => !!v}
               />
             </Tooltip>
-            <Tooltip content="Frist for avmelding – fører til prikk etterpå">
+            <Tooltip content="Frist for påmelding - antall timer før arrangementet">
+              <Field
+                key="registrationDeadlineHours"
+                label="Påmeldingsfrist"
+                name="registrationDeadlineHours"
+                type="number"
+                component={TextInput.Field}
+                fieldClassName={styles.metaField}
+                className={styles.formField}
+              />
+            </Tooltip>
+            <p className={styles.registrationDeadlineHours}>
+              <FormatTime
+                time={moment(event.startTime).subtract(
```

Comment:
If you add something like:

`registrationCloseTime: something` here: https://github.com/webkom/lego-webapp/blob/master/app/routes/events/EventEditRoute.js#L59

So that the `event.registrationCloseTime` is updated when changing the `registrationDeadlineHours`-field in the form. You can use `valueSelector(state, 'registrationDeadlineHours')` to get the selected variable, and calculate the new `event.registrationCloseTime`, and use that value directly here. (like you do in in `EventDetail/index.js`).